Packages/Fluxity/Runtime/Presenter.cs

```diff
@@ -15,25 +15,23 @@ public virtual void Start()
             Display();
         }
 
-        public abstract void Display();
-
         public abstract void CreateBindings();
 
-        public IFeaturePresenterBinding<TState> Bind<TState>()
+        public abstract void Display();
+
+        public FeatureBinding<TState> Bind<TState>()
```

Comment:
Suggest that this returns an IFeatureView<TState> rather than a FeatureBinding;

You need the state but you don't need access to the methods, also in the examples, some use IFeatureView, some use FeatureBinding.